### ISSUE TITLE
A11y_code.visualstudio.com_DownloadInsiders_ScreenReader: The name for “Download for Windows” link is inappropriate as compared to it’s visual appearance.

### ISSUE THREAD

**Issue by MayuriNaik** [microsoft/vscode-docs#2800]
“[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Faccessibilityinsights.io%2F&data=02%7C01%7Cv-manai%40microsoft.com%7Cb67b2c4b646d4f9561a208d6f4b5c39b%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636965458847260936&sdata=T26HQfSGOlnuRQdX%2ByXk%2B2bxqgwFvCIVfuboZUWidYY%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**Environment Details:**
URL: [https://code.visualstudio.com](https://code.visualstudio.com)
Windows Version: Windows10
Browser Version: 
Microsoft Edge 42.17134.1.0
Chrome Version 75.0.3770.100 (Official Build) (64-bit)
Screen Reader Versions : NVDA2019.1.1
MAS Violated: MAS 2.4.4

**Repro Steps:**
Navigate to the url: [https://code.visualstudio.com/insiders/](https://code.visualstudio.com/insiders/) 
follow below steps:
1.	Launch the application Url: [https://code.visualstudio.com ](https://code.visualstudio.com )
2.	Navigate to "Download" button.
3.	Scroll down the page and hit "Insiders build" link.
4.	Navigate through the "Download Insiders" screen and navigate to "Download for Windows 7, 8, 10 link".

**Actual:**
When screen reader users navigate to the "Download for Windows 7, 8, 10 link" it is announced as “Download for Windows” link which is incorrect as it provides incomplete information regarding the link.

The same issue is observed for "Download for Mac", ".deb" and "rpm"  elements

Similar issue is observed with "Edge + Narrator" as well as in "Chrome + NVDA".

**Expected:**
When screen reader users navigate to the “Download for Windows” link it should be announced as "Download for Windows 7, 8, 10 link" as per it's visual appearance.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
[https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx
](https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx
)
**User Impact:**
If the interactive element is not provided with proper name as per its visual appearance then the screen reader users will not get the complete information as visual users.

**MAS Reference:**
MAS 2.4.4 - [https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={06a891af-210c-4c6d-bb3e-92babab209e7}](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={06a891af-210c-4c6d-bb3e-92babab209e7}
)
**Attachment for Reference:**
![Code visualstudio com_DownloadInsiders_K4D Name property is inappropriate for link](https://user-images.githubusercontent.com/52036772/60016463-e5c15880-96a3-11e9-9436-7dbb73c6519b.png)
